Packs/CortexXpanse/Integrations/FeedXpanse/FeedXpanse.yml

```diff
@@ -113,6 +113,16 @@ configuration:
   section: Collect
   advanced: true
   required: false
+- display: Indicators types to fetch
+  name: indicatorTypes
+  type: 16
+  required: true
+  options:
+  - IP
+  - Domain
+  - Certificate
+  additionalinfo: 'Indicator types to fetch.'
+  section: Collect
 description: Use this feed to retrieve the discovered IPs/Domains/Certificates from Cortex Xpanse asset database.
```

Comment:
Use this feed to retrieve the discovered IPs/domains/certificates from the Cortex Xpanse asset database.